docs/docusaurus/docs/core/expectation_suites/manage_expectation_suites.md

```diff
@@ -0,0 +1,294 @@
+
+## Prerequisites
+
+This guide assumes you have:
+
+- Installed Python.
+- Installed the GX Core library.
+
+## Create an Expectation Suite
+
+1. Import the GX Core library and the `ExpectationSuite` class.
+  ```python name="core/expectation_suites/_examples/create_an_expectation_suite.py imports"
+  ```
+
+2. Get a Data Context.
+  ```python name="core/expectation_suites/_examples/create_an_expectation_suite.py get_context"
+  ```
+
+3. Create an Expectation Suite.
+  ```python name="core/expectation_suites/_examples/create_an_expectation_suite.py create Expectation Suite"
+  ```
+
+4. Add the Expectation Suite to your Data Context.
+  ```python name="core/expectation_suites/_examples/create_an_expectation_suite.py add snippet to Data Context"
+  ```
+
+:::tip
+You can add an Expectation Suite to your Data Context at the same time as you create the Expectation Suite with the following code:
+```python name="core/expectation_suites/_examples/create_an_expectation_suite.py create and add Expectation Suite to Data Context"
+```
+:::
+
+<details><summary>Full example code</summary>
+<p>
+
+```python name="core/expectation_suites/_examples/create_an_expectation_suite.py full example code"
+```
+
+</p>
+</details>
+
+## Get an existing Expectation Suite
+
+1. Import the GX Core library.
+  ```python name="core/expectation_suites/_examples/get_an_expectation_suite.py imports"
+  ```
+
+2. Get a Data Context.
+  ```python name="core/expectation_suites/_examples/get_an_expectation_suite.py get_context"
+  ```
+
+3. Use the Data Context to retrieve the existing Expectation Suite.
+  ```python name="core/expectation_suites/_examples/get_an_expectation_suite.py create Expectation Suite"
+  ```
+
+<details><summary>Full example code</summary>
+<p>
+
+```python name="core/expectation_suites/_examples/get_an_expectation_suite.py full example code"
+```
+
+</p>
+</details>
+
+## Edit an Expectation Suite
+1. [Create a new](#create-a-new-expectation-suite) or [get an existing](#get-an-existing-expectation-suite) Expectation Suite.  This example creates a new Expectation Suite which will then be renamed.
+  ```python name="core/expectation_suites/_examples/edit_an_expectation_suite.py create expectation suite"
+  ```
+
+2. Modify the Expectation Suite's attributes.
+  ```python name="core/expectation_suites/_examples/edit_an_expectation_suite.py edit attribute"
+  ```
+
+3. Save the Expectation Suite.
+  ```python name="core/expectation_suites/_examples/edit_an_expectation_suite.py save the Expectation"
+  ```
+
+  :::info  
+  The `save()` method of an Expectation Suite will also save any changes that have been made to Expectations contained by that Expectation Suite.
+  :::
+
+<details><summary>Full example code</summary>
+<p>
+
+```python name="core/expectation_suites/_examples/edit_an_expectation_suite.py full example code"
+```
+
+</p>
+</details>
+
+## Delete an Expectation Suite
+
+1. Import the GX Core library.
```

Comment:
```suggestion
1. Import the GX Core library.
```
Why is this the first step? Shouldn't users be instructed to get an existing Expectation Suite?

Reply:
Each of these examples is self-contained, so that there aren't any missing steps if the user skips to a particular procedure without looking at the previous ones.  Therefore, each needs to start with `import great_expectations as gx`.